activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -120,7 +120,7 @@ def create(*arguments)
         $stderr.puts "Database '#{configuration['database']}' already exists"
       rescue Exception => error
         $stderr.puts error
-        $stderr.puts "Couldn't create database for #{configuration.inspect}"
+        $stderr.puts "Couldn't create database for #{configuration['database']}"
```

Comment:
Should the text be changed to something like `"Couldn't create database for #{configuration['database']}. Please check your configuration."` so its clearer that the database config is the issue?